definitions/npm/react-dnd_v2.x.x/flow_v0.53.x-/test_react-dnd-v2.js

```diff
@@ -288,7 +285,6 @@ const dndKnight3: DndKnight3 = (null: any);
   typeof DndKnight3
 >);
 
-// $ExpectError: Invalid collect prop
```

Comment:
@Stephen2 why these are removed?

Reply:
Tests failed.  I suppose these things no longer fail, so $ExpectError was wrong now?  I don't really understand the testing, but removing these fixed failing tests